testfm/constants.py

```diff
@@ -1,17 +1,25 @@
-import configparser
+from dynaconf import LazySettings
 
-config = configparser.ConfigParser()
-config.read("testfm.properties")
+settings = LazySettings(
+    envvar_prefix="TesTFM",
+    core_loaders=["YAML"],
+    settings_file=["settings.yaml"],
+    preload=["*.yaml"],
+    includes=["settings.local.yaml", ".secrets.yaml", ".secrets_*.yaml"],
+    envless_mode=True,
```

Comment:
If you want to access settings from environment variables via OCP or via Jenkins Job/Pipeline then only set envless mode else not required.

Reply:
Yes, it is required for the pipeline.